packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -391,6 +394,7 @@ Your $application code is in $relativeAppMain.
       templateContext['linux'] = false;
       templateContext['macos'] = false;
       templateContext['windows'] = false;
+      templateContext['winuwp'] = false;
```

Comment:
Maybe a list of platforms would help clean this up, like:
```dart
for (final String platform in kPlatforms) {
  templateContext[platform] = false;
}
```

Reply:
done